content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -138,15 +138,12 @@ Provisioner.
 
 ### Minimum ready seconds
 
-{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
 
 `.spec.minReadySeconds` is an optional field that specifies the minimum number of seconds for which a newly
 created Pod should be ready without any of its containers crashing, for it to be considered available.
-Please note that this feature is beta and enabled by default. Please opt out by unsetting the
-StatefulSetMinReadySeconds flag, if you don't
-want this feature to be enabled. This field defaults to 0 (the Pod will be considered 
-available as soon as it is ready). To learn more about when a Pod is considered ready, see
-[Container Probes](/docs/concepts/workloads/pods/pod-lifecycle/#container-probes).
+This defaults to 0 (the Pod will be considered available as soon as it is ready). To learn more about when 
+a Pod is considered ready, see [Container Probes](/docs/concepts/workloads/pods/pod-lifecycle/#container-probes).
 
```

Comment:
I would make it clear that `minReadySeconds` is really there to phase the rollout of new or updated StatefulSets (whereas probe thresholds are the right thing to use if you want to, eg, see that the Pod has had a healthcheck every 2 seconds and that there have been 30 successful ones, which implies that the Pod is ready for traffic).

Reply:
Good suggestion, but we also say that here https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#rolling-updates. 

Please let me know if you are satisfied with the current state.

Reply:
I'd like to repeat that detail under the Minimum Ready Seconds heading, in case people didn't arrive via https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#rolling-updates.

Reply:
What do you think about having a link to the Rolling Updates so we do not have to define the strategy again? commit updated..

Reply:
@sftim -- do you still have concerns, or can we approve this?

Reply:
@sftim looks like this was merged without the comment being addressed, do we need any further followup?

Reply:
It'd be nice to fix that, but that fix can happen after v1.25 ships.

Reply:
> What do you think about having a link to the Rolling Updates so we do not have to define the strategy again?

I can post an update, but what are problems with this approach? ^